test/e2e/performanceprofile/functests/utils/namespaces/namespaces.go

```diff
@@ -30,6 +30,12 @@ func init() {
 var TestingNamespace = &corev1.Namespace{
 	ObjectMeta: metav1.ObjectMeta{
 		Name: testutils.NamespaceTesting,
+		Labels: map[string]string{
```

Comment:
that's not the original commit, would have been (much much) better to note that in the commit message, mentioning the other one squashed here

Reply:
something like:
```
e2e: add podsecurity labels, disable OCP label sync

Backport and squash two commits related to handling
of security labels:

b0024584fb244691097916665d5cec1e7c1dc5f4 (partial):    
    OCP >= 4.12 wants to have stricter podsecurity rules.
    In our e2e tests we do a bunch of stuff, including running privileged
    pods. We just annotate the test namespace(s) to signal we need top
    privileges. Since e2e tests run in very controlled envs (CI mostly),
    and since test namespace should be gone anyway once the e2e tests
    are finished, this is still fair game.
    
    Signed-off-by: Francesco Romani <fromani@redhat.com>

b0024584fb244691097916665d5cec1e7c1dc5f4
    podsecurity: disable OCP label sync

    Per last OCP recommendation (internal doc) is not sufficient to
    declare the pod security labels, we also need to disable the
    label sync, which we do here.

    Signed-off-by: Francesco Romani <fromani@redhat.com>
    
Co-Authored-by: Mario Fernandez <mariofer@redhat.com>
Signed-off-by: Francesco Romani <fromani@redhat.com>
```
(please doublecheck the commit hashes)

Reply:
thanks for fixing the PR description. It would have been much better to fix the relevant commit message AND THEN the PR description, but I guess we can move forward this time. Let's evaluate deeper next time.

Reply:
I can do it but whats is the purpose of that if we squash the commits?

Reply:
we should not squash the commits when doing backports - we should have as much as possible the same commits as the main branch, otherwise half of the purpose of doing backports is lost.
In general, we should preserve the authorship properly and the commit boundary as much as possible. There's almost never a good reason not to.
That becomes critical if contributions are from different organizations.

Reply:
Get it.

Reply:
Folks, the tests passed, @fromanirh can you add lgtm if all is fine now?

Reply:
> Folks, the tests passed, @fromanirh can you add lgtm if all is fine now?

I was waiting for other reviews.

Reply:
@yanirq @swatisehgal PTAL?